api/src/db/config/dynamoDbConfig.ts

```diff
@@ -22,11 +23,23 @@ export const unmarshallOptions = {
 
 export const dynamoDbTranslateConfig = { marshallOptions, unmarshallOptions };
 
-export const migrateData = (data: any): any => {
+export const migrateData = (data: any, logger: LogWriterType): any => {
```

Comment:
Pebble - I think this function shuould continue to live within `DynamoUserDataClient` - I think it's the only file it should be used in. I left a comment in `DynamoBusinessDataClient` the migration fn can only be used on userData, not the business object.

Reply:
This is going back since its not reusable.